src/Logging/Logging.Console/src/ConsoleLogger.cs

```diff
@@ -73,53 +73,121 @@ public virtual void WriteMessage(LogLevel logLevel, string logName, int eventId,
                 logBuilder = new StringBuilder();
             }
 
-            // Example:
-            // INFO: ConsoleApp.Program[10]
-            //       Request received
-
-            var logLevelColors = GetLogLevelConsoleColors(logLevel);
-            var logLevelString = GetLogLevelString(logLevel);
-            // category and event id
-            logBuilder.Append(_loglevelPadding);
-            logBuilder.Append(logName);
-            logBuilder.Append("[");
-            logBuilder.Append(eventId);
-            logBuilder.AppendLine("]");
-
-            // scope information
-            GetScopeInformation(logBuilder);
-
-            if (!string.IsNullOrEmpty(message))
+            if (Options.Format == ConsoleLoggerFormat.Default)
             {
-                // message
-                logBuilder.Append(_messagePadding);
+                // Example:
+                // INFO: ConsoleApp.Program[10]
+                //       Request received
+
+                var logLevelColors = GetLogLevelConsoleColors(logLevel);
```

Comment:
Could we put the two formats into separate methods? i.e. `BuildDefaultMessage`/`BuildSystemdMessage`. Have each method return a `LogMessageEntry` and the outer code queues it.